pkg/controllers/consolidation/helpers.go

```diff
@@ -67,3 +76,42 @@ func disruptionCost(ctx context.Context, pods []*v1.Pod) float64 {
 	}
 	return cost
 }
+
+// getNodePrice gets the last known node price for the candidate node
+// This price is used to determine the savings that can be achieved through consolidation
+func getNodePrice(node candidateNode) (float64, error) {
```

Comment:
How is this method different than just calling `cloudprovider.GetOffering(node.instanceType.Offerings(), node.capacityType, node.zone)` directly?